SOPT-iOS/Projects/Features/DailySoptuneFeature/Sources/DailySoptuneScene/Views/DailySoptuneResultPokeView.swift

```diff
@@ -12,14 +12,17 @@ import Combine
 import DSKit
 import Core
 import Domain
-import PokeFeature
 
 public final class DailySoptuneResultPokeView: UIView {
     
     public lazy var kokButtonTap: Driver<PokeUserModel?> = kokButton.tap
         .map { self.user }
         .asDriver()
     
+    lazy var profileTap = self.profileImageView.gesture()
+        .filter { _ in self.user?.isAnonymous == false }
+        .map { _ in self.user }.asDriver()
```

Comment:
```suggestion
        .filter { [weak self] _ in self?.user?.isAnonymous == false }
        .map { [weak self] _ in self?.user }.asDriver()
```

오퍼레이터들도 escaping 클로저라 self를 참조할 때 약한 참조를 사용해야 누수를 방지할 수 있어요.

Reply:
깜짝.. 등장해서 .. 리뷰.. 해주시는 ..건가요 ...
감사합니다 ,,,,,,,